sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GalleryApplicationVersionPublishingProfile.cs

```diff
@@ -78,11 +75,9 @@ public GalleryApplicationVersionPublishingProfile(UserArtifactSource source, ILi
         public UserArtifactSource Source { get; set; }
 
         /// <summary>
-        /// Gets or sets optional. May be used to help process this file. The
-        /// type of file contained in the source, e.g. zip, json, etc.
         /// </summary>
-        [JsonProperty(PropertyName = "contentType")]
-        public string ContentType { get; set; }
+        [JsonProperty(PropertyName = "manageActions")]
+        public UserArtifactManage ManageActions { get; set; }
```

Comment:
This is a breaking change - you will have to maintain a ContentType property, even if unused